source/reference/template-apps.txt

```diff
@@ -116,8 +116,8 @@ with services and front-end code. You can:
 
 .. tip::
 
-   All the template apps except "Manage Database Views" come with a
-   front-end todo app.
+   All of the available template apps except "Manage Database Views" come with a
+   front-end to-do list app.
```

Comment:
can we cut this line? 

i don't think this is true anymore b/c of the templates in  https://www.mongodb.com/docs/atlas/app-services/sync/data-access-patterns/flexible-sync-permissions-guide/

also in the future we def might make templates that are backend only.

more maintainable to remove.

Reply:
I'll defer to you on this, though I figured this was noted because the 'Manage DB Views' template app is the only one that doesn't include any frontend code for you to pull.